lib/generators/capistrano/sidekiq/systemd/templates/sidekiq.service.capistrano.erb

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=sidekiq for <%= "#{fetch(:application)} (#{fetch(:stage)})" %>
+After=syslog.target network.target
+
+[Service]
+Type=simple
+Environment=RAILS_ENV=<%= fetch(:rails_env) %>
+WorkingDirectory=<%= fetch(:deploy_to) %>/current
+ExecStart=<%= fetch(:bundler_path, '/usr/local/bin/bundler') %> exec sidekiq -e <%= fetch(:rails_env) %>
+ExecReload=/bin/kill -USR1 $MAINPID
```

Comment:
It looks like this should be `ExecReload=/bin/kill -TSTP $MAINPID` (TSTP instead of USR1) for Sidekiq 5. https://github.com/mperham/sidekiq/wiki/Signals#tstp

Reply:
Sure.